api/agent/call.go

```diff
@@ -135,6 +136,17 @@ func buildConfig(app *models.App, route *models.Route) models.Config {
 	return conf
 }
 
+func buildAnnotations(app *models.App, route *models.Route) models.Annotations {
+	ann := make(models.Annotations)
```

Comment:
tip: can pre-allocate the size perfectly here `make(models.Annotations, len(app.Annotations)+len(route.Annotations)` to avoid some number of map resizes (~$)